hf-example/index.html

```diff
@@ -0,0 +1,162 @@
+                      </optgroup>
+                      <optgroup label="Pryv.io">
+                        <option value="https://reg.pryv.me/service/info"
+                                selected="selected">Pryv Lab</option>
+                        <option value="https://reg.{DOMAIN}/service/info">Your own domain</option>
+                      </optgroup>
+                    </select>
+                        <div class="form-group">
+                            <label for="service-info">Service Information URL:</label>
+                            <input type="text" class="form-control" id="service-info-text" value="https://reg.pryv.me/service/info" />
+                        </div>
+
+                    </div>
+                    <br />
+                    <div class="spaced">
+                        <button type="button" id="fetch-service-info-button" class="btn btn-primary mb-2">Fetch service
+              info</button>
+                        <br>
+                        <span id="pryv-button"></span>
+                    </div>
+                </div>
+            </div>
+
+            <div class="card" id="mouse-tracker">
+                <div class="card-body">
+                    <h2 class="card-title">Mouse Tracker</h2>
+                    <br/>
+                    <p>You can select the image of your choice in the selector. If you change the image, it will erease your drawing on the screen. You need to left click to begin and to stop to draw.</p>
+                    <br/>
+                    <select name="image-selector" id="image-selector">
+                  <option value="images/heart.png">Heart</option>
+                  <option value="images/house.png">House</option>
+                </select>
+                    <div id="drawing-field">
+                        <canvas class="canvas" id="drawing-canvas" height="400px" width="600px"></canvas>
+                    </div>
+
+
+                </div>
+            </div>
+
+
+            <div class="card" id="mouse-visualization">
+                <div class="card-body">
+                    <h2 class="card-title">Mouse Visualization</h2>
+                    <br />
+                    <p>Click on previous/next to see the saved images. Click on current to see the current drawing.</p>
+                    <br />
+                    <span id="frequency-mouse"></span>
+                    <br/>
+                    <div id="canvas-container">
+                        <canvas class="canvas" id="render-canvas" height="400px" width="600px"></canvas>
+                        <div id="image-button">
+                            <button id="previous" class="image-button">Previous</button>
+                            <button id="current" class="image-button">Current</button>
+                            <button id="next" class="image-button">Next</button>
+                        </div>
```

Comment:
We need to add a link to a previous web app "Next example: View and Share Data" (https://github.com/pryv/app-web-examples/tree/master/view-and-share)

Reply:
Fixed